drivers/FileHandle.h

```diff
@@ -91,7 +114,8 @@ class FileHandle {
      *    new file position on success,
      *    -1 on failure or unsupported
      */
-    virtual off_t lseek(off_t offset, int whence) = 0;
+    MBED_DEPRECATED_SINCE("mbed-os-5.4", "Replaced by FileLike::seek")
+    virtual off_t lseek(off_t offset, int whence) { return seek(offset, whence); }
```

Comment:
Why seek rather than lseek? lseek is the standard POSIX name - it struck me as good to use standard names to indicate standard intent.